scss/_utilities.scss

```diff
@@ -27,6 +27,7 @@ $utilities: map-merge(
     ),
     "shadow": (
       property: box-shadow,
+      hover: $enable-hover-utilites,
```

Comment:
In this case I will generate `hover` together with simple state. What if I need to generate only `hover` utilities? And why only `hover`? Maybe I need `focus`? 
I would say declaration should be via additional property which will generate only for this property:

`state: "hover"`

full declaration:

```
 "shadow-hover": (
    property: box-shadow,
    state: hover,
    class: shadow,
    values: (...)
)
```

Reply:
Yes, at first I did it (you can see the old pr), but then I thought that it might be a complication. We need some kind of default state that describes classes that will not affect the box-model

Reply:
In fact, this is how it works now. By removing the hover field, you disable class generation for a specific property

Reply:
Is this about this PR https://github.com/twbs/bootstrap/pull/31640/files ? I see it's almost the same, just $enable-hover-utilites added

Reply:
Seems usage of `state` / `variant` instead of `hover: true` is more intuitive and flexible according to this issue: https://github.com/twbs/bootstrap/issues/31856

Reply:
Yes, that's a good idea. thanks